runtime/doc/vvars.txt

```diff
@@ -727,6 +727,19 @@ v:version
 		  :if has("nvim-0.2.1")
 <
 
+				*v:versionlong* *versionlong-variable*
+v:versionlong
```

Comment:
Do we actually need this? Not a blocker but seems kind of useless.

Reply:
I personally still use `v:version`  so I _may_ (?) migrate to it in the future.

It can be misleading compared to `has('patch-a.b.c')` but it's easier for users who just want to know the highest patch number supported for their configs without resorting  to `:version` or `nvim -v`.